src/app/mocks/products.ts

```diff
@@ -0,0 +1,95 @@
+    picture: 'https://petsi.net/images/dogbreed/26.jpg',
+  },
+];
+
+const PRODUCTS_MOCK_EN: Array<Product> = [
+  {
+    name: 'Australian shepherd',
+    description: `The Australian Shepherd, Aussie is a medium-sized ranch bred in the western states of the United States. Despite the name, they are not related to Australia, their homeland is America.`,
+    picture: 'https://www.royal-canin.ru/upload/iblock/117/avstr.ovcharka2.jpg',
+  },
+  {
+    name: 'Australian cattle dog',
+    description:
+      'The Australian Cattle Dog breed originally originated in Australia. A herding dog who helped drive herds across the harsh lands. Medium in size and short-haired, they come in two colors - blue and red.',
+    picture:
+      'https://lh3.googleusercontent.com/proxy/1DNxAKCTIoz2T_tHbSXu8A7xzdAmWAZX5ini_f4Jg9aOsoVXSaHnEVZnWrtbkgWL8dyg3Oqe2c0uEvuZyEDpC_AeRqlpuGN2Ke5P2SNDjVwAeUQ',
+  },
+  {
+    name: 'Australian kelpie',
+    description:
+      'The Australian Kelpie is a herding dog native to Australia that is adept at handling a herd without the help of an owner. Medium in size, it can be of almost any color and is now mostly used for its intended purpose.',
+    picture:
+      'https://lh3.googleusercontent.com/proxy/Luu_RtvtnkAXfEOwLV6HIJqlwGeErNaoPee0ot41bCEBwxQ9Z505hkaUfwCTPHzFRkIs04QS5kmrXn4',
+  },
+  {
+    name: 'Alaskan Malamute',
+    description:
+      'The Alaskan Malamute is a large sled dog in Alaska. It is believed that this is one of the oldest dog breeds, bred by the Eskimo tribe Malemute, first as an ordinary dog, and then as a sled dog. They are often confused with Siberian huskies, as they have similar colors, but malamutes are much larger and have a more dominant character.',
+    picture:
+      'https://mybarbos.com/wp-content/uploads/2016/08/aljaskinskij-malamut-03.jpg',
+  },
+  {
+    name: 'American bully',
+    description:
+      'The American Bully is a young dog breed that first appeared in the 1990s and suddenly became very popular. These dogs are known for their harsh and intimidating appearance but friendly disposition.',
+    picture: 'https://sobaky.info/wp-content/uploads/2020/04/7-7.jpg',
+  },
+  {
+    name: 'Basenji',
+    description:
+      'Basenji or African barking dog (English Basenji) is the oldest breed of hunting dogs, originally from central Africa. These dogs make unusual rumbling sounds as they have an unusual larynx shape. For this they are also called not barking dogs, but the sounds they make are “barroo”.',
+    picture: 'https://petsi.net/images/dogbreed/26.jpg',
+  },
+];
+
+export function getProducts(lang: Lang): Array<Product> {
+  switch (lang) {
+    case 'en':
+      return PRODUCTS_MOCK_EN;
+    case 'ru':
+      return PRODUCTS_MOCK_RU;
+  }
+
+  return [];
```

Comment:
This is not resolved. ! Don't forget to case enums instead of pure strings.

Reply:
`Move this to the switch body under default case.` - It would behave the same way. So what is the difference ? What benefits it gives if I would move it there?

Reply:
According to the GCode style 
_**All switch statements must contain a default statement group, even if it contains no code.**_